cmd/cluster/aws/destroy.go

```diff
@@ -21,9 +22,10 @@ func NewDestroyCommand(opts *core.DestroyOptions) *cobra.Command {
 	}
 
 	opts.AWSPlatform = core.AWSPlatformDestroyOptions{
-		AWSCredentialsFile: "",
-		PreserveIAM:        false,
-		Region:             "us-east-1",
+		AWSCredentialsFile:  "",
+		PreserveIAM:         false,
+		Region:              "us-east-1",
+		AwsInfraGracePeriod: 30 * time.Minute,
```

Comment:
why are changing default? can we please keep current behaviour and let this be opt-int?

Reply:
addressed with [commit](https://github.com/openshift/hypershift/pull/2967/commits/4a34b10409ac327d80bb1cc62c28e2f224bcac1f)